lib/rowFactory.js

```diff
@@ -36,7 +36,8 @@ module.exports = {
                 stateManager: options.stateManager,
                 treeColumn: options.treeMode && index === 0,
                 level: level,
-                treeState: treeState
+                treeState: treeState,
+                launchLinksNewTab: options.launchLinksNewTab || false
```

Comment:
Wrong place to default.  Should be done in grid.js with all the other defaults.  The default should also be tested with all the other defaults in gridSpec.js